kumascript/src/api/wiki.js

```diff
@@ -69,7 +69,7 @@ module.exports = {
       return html;
     }
 
-    let result = this.info.getResultFromCache(path);
+    let result = await this.renderFromURL(path);
     const pathDescription = this.info.getDescription(path);
 
     if (!result) {
```

Comment:
With your new approach, this entire `if` block is no longer needed.

Reply:
Awesome, removed! Thanks